modules/core/src/lib/api/load.js

```diff
@@ -13,20 +13,25 @@ export async function load(url, loaders, options) {
     loaders = null;
   }
 
+  // Normalize arguments
+  options = options || {};
+
+  // Select fetch function (Note: binds options, if supplied)
+  const fetch = getFetch(options);
```

Comment:
`options` here is not merged with `defaultOptions`. Are we phasing out global options?

Reply:
Great catch, completely missed that. Took a second look and the current options handling is too messy. It might be best to make a prep diff that cleans up options handling before updating this PR to make sure things are more clear and maintainable, I'll update here before landing.